kafka/README.md

```diff
@@ -1,6 +1,15 @@
 # Information for confluent-kafka-go developers
 
-Whenever librdkafka error codes are updated make sure to run generate
+## Development process
+
+1. Use go1.18 (and related tooling) for development on confluent-kafka-go.
```

Comment:
Instead of "Use go1.18 (and related tooling)", we can write "Use Go (preferred v1.18+) and related tooling"

Reply:
We don't want people to use 1.19, because gofmt in that version is not backwards compatible. We will need to make one gofmt commit before we are able to encourage people to use 1.19.

As for versions before 1.18, they would work, except that they are no longer supported by golang developers.